src/domain/cartItem.ts

```
@@ -0,0 +1,10 @@
+import { CartItemType } from '../types/data';
+
+export const getCartId = async (cartItems: CartItemType[], productId: number) => {
+  const targetItem = cartItems.find((item: CartItemType) => item.product.id === productId);
+  return targetItem && targetItem.id;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)